mirage/unikernel.ml

```diff
@@ -268,7 +450,27 @@ module Main (N : Mirage_net.S) = struct
             (Mirage_mtime.elapsed_ns ())
             Mirage_crypto_rng.generate primary_t
         in
-        Resolver.resolver stack ~root:true resolver;
+        let ca = CA.make "robur.coop" (Base64.encode_exn "foo") in
```

Comment:
Three questions appears here:
1) the domain-name of the ephemeral CA that we generate should be specified by the user?
2) the seed to generate the RSA private key should also be specified by the user? (a tool should also be provided to generate the same certificate)
3) the certificate is valid only for one year, we should make a loop to regenerate the CA